static/js/components/TopAppBar_test.js

```diff
@@ -13,7 +15,7 @@ describe("TopAppBar component", () => {
     const user = makeAnonymousUser()
     it("has a link to login", () => {
       assert.equal(
-        shallow(<TopAppBar currentUser={user} location={null} />)
+        shallow(<AnonymousMenu useScreenOverlay={false} />)
```

Comment:
You're just rendering the `AnonymousMenu` component here, so this is no longer a `TopAppBar` test case and should be moved to `AnonymousMenu_test.js`. Same comment for lines 28, 66, and 74